spec/card_generator_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './lib/card_generator'
+
+RSpec.describe CardGenerator do
+  describe 'initialize' do
+    it 'creates and instance of CardGenerator' do
+      filename = cards.txt
```

Comment:
`cards.txt` will need to be a string, whenever you come back to finish this out.